### PR TITLE
chore(gatsby,gatsby-core-utils): Upgrade lmdb to 2.5.2

### PR DESCRIPTION
**Pull request by wardpeet**
## Description

Upgrade lmdb to latest version

### CODE REVIEW

File: packages/gatsby-core-utils/package.json [hunk 1]
```diff
@@ -50,7 +50,7 @@
     "fs-extra": "^10.1.0",
     "got": "^11.8.3",
     "import-from": "^4.0.0",
-    "lmdb": "2.3.10",
+    "lmdb": "2.4.2",
```
Comment (LekoArts) [inline packages/gatsby-core-utils/package.json:53]: Typo?

Comment (LekoArts) [inline packages/gatsby-core-utils/package.json:53]: Typo?

Comment (wardpeet) [reply packages/gatsby-core-utils/package.json:53]: what do you mean with typo? I can't see it

Comment (LekoArts) [reply packages/gatsby-core-utils/package.json:53]: lol, scratch that. I read 2.3.2 🤣

**Review by pieh — APPROVED** [review]

**Pull request merged**